misc/protobuf-ballerina/src/main/java/org/ballerinalang/protobuf/utils/BalFileGenerationUtils.java

```diff
@@ -0,0 +1,203 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.protobuf.utils;
+
+import com.google.protobuf.DescriptorProtos;
+import org.ballerinalang.protobuf.exception.BalGenToolException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UnsupportedEncodingException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Locale;
+
+/**
+ * Util function used when generating bal file from .proto definition.
+ */
+public class BalFileGenerationUtils {
+    private static final Logger LOG = LoggerFactory.getLogger(BalFileGenerationUtils.class);
```

Comment:
May be you could use a consistent variable name for this loggers. Noticed this PR used several variable names for these loggers.